src/dispatch/incident/flows.py

```diff
@@ -630,25 +630,22 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
     )
 
 
-@background_task
-def incident_active_flow(incident_id: int, command: Optional[dict] = None, db_session=None):
+def incident_active_status_flow(incident: Incident, db_session=None):
     """Runs the incident active flow."""
-    # we load the incident instance
-    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
-
-    # we remind the incident commander to write a tactical report
-    send_incident_report_reminder(incident, ReportTypes.tactical_report, db_session)
```

Comment:
I think we should keep the reminder, as we want the IC to inform everyone about why it was re-opened?

Reply:
In the case of going from "closed" -> "stable" we run the active and then stable flows. Since stable is already sending a reminder, I wanted to avoid duplicate reminders.

If we want this reminder for all status changes except "closed" I can move it out of a specific flow.

Reply:
Yeah, I think it makes sense to send the reminder for both active and stable.